src/app/shell/personal-cabinet/shared-cabinet/applications/application-card/application-card.component.scss

```diff
@@ -10,13 +10,24 @@
     display: flex;
     justify-content: space-between;
 
-    & > div {
+    &>div {
```

Comment:
Please keep the previous formatting for the selectors, it makes them more readable

Reply:
Made it